src/dialogs/PaymentErrorDialog.vue

```diff
@@ -28,19 +28,20 @@
         <!-- display errors -->
         <div class="font-15 mb-4" v-if="numErrors > 0">
           <p>We were unable to process your payment due to the following errors:</p>
-          <ul>
-            <li v-for="(error, index) in errors" :key="index">
-              {{error.message}}
-            </li>
-          </ul>
+          <div v-for="(error, index) in errors" :key="index" class="d-flex">
+            <span id="error-bullet" class="flex-shrink-0">
+              <v-icon id="error-circle">mdi-circle-medium</v-icon>
```

Comment:
As this template is in a for loop, it may result in duplicate IDs, which is invalid on a HTML page.

Reply:
I will use class instead.

Reply:
Fixed.